Bottom: a715fe105d067edef9d1ab0d4416f284a09f605f
Top:    d6b6821caf97a60fad45a27cf1cb4be567fa05eb
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-08-05 14:14:54 -0400

copy


---

diff --git a/cop b/cop
new file mode 100755
index 0000000..42d5878
--- /dev/null
+++ b/cop
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+
+make zImage uImage dtbs
+#cp /work/hans/sound/soc/sunxi/sunxi-codec.ko /work/root/lib/modules/3.15.0+/kernel/sound/soc/sunxi
+
